com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,95 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+    private const val EMPTY_STRING = ""
+
+    @JvmStatic
+    fun serializeHeliumEvent(
+        placement: String?,
+        heliumAdError: HeliumAdError?,
+        eventConsumer: HeliumEventConsumer<String?, Int, String?>
+    ) {
+        var placementName: String? = EMPTY_STRING
+        if (placement != null) placementName = placement
+        var errorCode = -1
+        var errorDescription = EMPTY_STRING
+        if (heliumAdError != null) {
+            val tempCode = heliumAdError.getCode()
+            errorDescription = heliumAdError.getMessage()
+
+            // we do this in order to bypass lint issues
+            if (errorCode != tempCode) errorCode = tempCode
+        }
+        eventConsumer.accept(placementName, errorCode, errorDescription)
+    }
+
+    @JvmStatic
+    fun serializeHeliumBidEvent(
+        placement: String?,
+        dataMap: HashMap<String, String>,
+        eventConsumer: HeliumBidEventConsumer<String?, String?, String?, Double>
+    ) {
+        try {
+            var placementName: String? = EMPTY_STRING
```

Comment:
you're treating a lot of these as non null. You can update the signature of `eventConsumer: HeliumBidEventConsumer<String, String, String, Double>` to use not nullable ones.

Reply:
Yes, I noticed and cleaned this up afterwards :)